src/app/services/books.service.js

```diff
@@ -0,0 +1,19 @@
+angular.module('librarium')
+  .factory('BooksService',
+    ['Restangular', function (Restangular) {
+      const urlBase = '/books';
+      const booksService = {};
+
+      booksService.getBooks = function (attributes = {}) {
+        let searchParams = {};
+
+        attributes.page === undefined ? 1 : attributes.page;
```

Comment:
I was sending an empty page attribute. I should have written it this way:
`attributes.page === undefined ? attributes.page = 1 : attributes.page;`
I changed it to an if because since I changed to code to merge the two functions, now I only need to do one assignment.